### PR TITLE
app-admin/gtkdiskfree: version bump, move to EAPI 7

### PR DESCRIPTION
**Pull request by henrikp**
Closes: https://bugs.gentoo.org/665026
Package-Manager: Portage-3.0.1, Repoman-2.3.23
Signed-off-by: Henrik Pihl <ahvenas@gmail.com>

### CODE REVIEW

File: app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	!gtk2? ( x11-libs/gtk+:3 )
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf"
+RDEPEND="${DEPEND}"
+BDEPEND="sys-devel/gettext
+	virtual/pkgconfig"
+
+src_configure() {
+	econf $(use_with gtk2) \
+		$(use gtk2 || use_enable old-color-selector)
```
Comment (juippis) [inline app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild:27]: I wouldn't introduce a new USE flag for this. You can simply control it with `gtk2` IUSE.

It's also a bit weird that `-gtk2` pulls in gtk3, but uses something called **old**-color-selector?

File: app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="gtk2 old-color-selector"
+
+DEPEND="
+	dev-libs/glib
+	gtk2? ( x11-libs/gtk+:2 )
+	!gtk2? ( x11-libs/gtk+:3 )
```
Comment (juippis) [inline app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild:18]: Please sort this list by putting all conditional deps below unconditional ones, 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Sorting_dependencies

Comment (henrikp) [reply app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild:18]: Done.

File: app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild (hunk 1 above)
Comment (henrikp) [reply app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild:27]: The old color selector is superior than the one used with the GTK3 version, don't know why it is either.

**Pull request closed without merging**